$NetBSD: patch-aa,v 1.2 2020/02/04 17:25:59 rillig Exp $

Honor CFLAGS from pkgsrc. Not reported upstream since the project has not
had any activity since 2008.

--- Makefile.orig	Sat Sep  2 17:47:37 2000
+++ Makefile	Sat Sep  2 17:48:38 2000
@@ -13,11 +13,14 @@
 #CC=gcc -Wall -g
 #LEX=flex
 
+CC ?= gcc
+CFLAGS ?= -O
+
 pscan: scanner.yy.o pscan.o
 	$(CC) scanner.yy.o pscan.o -o pscan
 
 scanner.yy.o: scanner.yy.c
-	$(CC) -c scanner.yy.c -o scanner.yy.o
+	$(CC) $(CFLAGS) -c scanner.yy.c -o scanner.yy.o
 
 scanner.yy.c: scanner.l
 	$(LEX) -t scanner.l > scanner.yy.c
